living/buginese.md

```diff
@@ -1,16 +1,17 @@
 # Web Language: Buginese
-
+The Bugis language is primarily spoken by ~5 million people, mainly in South Sulawesi, Indonesia, with communities also present in Malaysia and Singapore.
 Additional names:
-- Basa Ugi
-- 
+- Basa Ugi ᨅᨔᨕᨘᨁᨗ
```

Comment:
Can the Lontara representations be on separate lines please? It would also be worth including "Bugis" here on another line.